packages/react-aria-components/src/Tabs.tsx

```diff
@@ -302,9 +311,10 @@ function TabPanel(props: TabPanelProps, forwardedRef: ForwardedRef<HTMLDivElemen
     <div
       {...domProps}
       ref={ref}
+      data-focused={isFocused || undefined}
       data-focus-visible={isFocusVisible || undefined}
       // @ts-ignore
-      inert={!isSelected ? 'true' : undefined} />
+      data-inert={!isSelected ? 'true' : undefined} />
```

Comment:
We still need the `inert` attribute for its native behavior, but we can document `data-inert` as the one to use for styling.

```suggestion
      inert={!isSelected ? 'true' : undefined}
      data-inert={!isSelected ? 'true' : undefined} />
```